styles.css

```diff
@@ -31,312 +31,685 @@
+    background: #ebf0ee;
+  }
+
+  /* Contact Form */
+  .footer {
+    height: 785px;
+    background-image: url("images/illustration-contact-form.svg");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .contact-form-control {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 40px;
+  }
+
+  .user-name {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+  }
+
+  .user-email {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+  }
+
+  .user-msg {
+    font-family: 'Inter', sans-serif;
+    border: 1px solid #d0d9d4;
+    width: 327px;
+    height: 114px;
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding-left: 10px;
+    padding-top: 10px;
+  }
+
+  .contact-for-contact-info {
+    margin-bottom: 40px;
+  }
+
+  .portfolio-owner-email {
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    height: 0;
+    color: #42526e;
+  }
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  .container {
+    padding-left: 20px;
+  }
+
+  .btn {
+    background-color: #36b37f;
+    padding: 17px 5px;
+    color: white;
+    border-radius: 5px;
+    font-family: 'Inter', sans-serif;
+    align-self: center;
+  }
+
+  /* Nav Bar */
+  .navbar-items {
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .menu-button {
+    display: none;
+  }
+
+  .Nav-Item {
+    margin-right: 30px;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    background-image: url("images/Header-llustration-desktop.svg");
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 500px;
+    padding: 0 24px;
+  }
+
+  .headline h1 {
+    font-family: 'Inter', sans-serif;
+    font-size: 40px;
+    color: #172b4d;
+    width: 100%;
+    margin-bottom: 24px;
+    padding: 0 12px;
+  }
+
+  .headline span {
+    font-family: 'Inter', sans-serif;
+    font-size: 40px;
+    color: #36b37e;
+  }
+
+  .headline p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    color: #42526e;
+    width: 100%;
+    margin-bottom: 24px;
+    padding: 0 100px;
+  }
+
+  .contact-info {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 30px;
+    margin-bottom: 50px;
+  }
+
+  .contact-info-button {
+    margin-right: 30px;
+  }
+
+  .disabled-button {
+    margin-top: 100px;
+  }
+
+  .generic {
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+  }
```

Comment:
- [ ] I really appreciate your design, you did great work so far, however, please implement these few changes and you will be good to go, see this image below, your paragraph text alignment is not the same as on the original design :arrow_down:. you can use `text-align: left` to fix this

![333333](https://user-images.githubusercontent.com/57562869/160001035-efdc714f-bb43-4c70-86f5-f5930a8b872c.png)


- [ IT SHOULD LOOK LIKE THIS INSTEAD]
<img width="453" alt="fig" src="https://user-images.githubusercontent.com/79058364/159996099-262774c4-c3a9-4483-8b55-025fc4f7b597.png">

Reply:
@moise10r I just made the changes to include in my last code review (`https://github.com/jrosario19/Juan-Rosario-Portfolio/pull/5`), and those requested changes were not in the first code review and now I'm afraid to lose my last attempts if another code reviewer notices other changes. Kindly, could you help me with that issue, because I think I'm able to continue to the next week?

Reply:
hey @jrosario19

Reply:
Yes

Reply:
Alright, you can go and request a review now!!